microsoft-edge/webview2/code-samples-links.md

```diff
@@ -1,213 +1,49 @@
 ---
-title: Sample Code for WebView2
-description: Links to WebView2 sample code hosted on GitHub.
+title: Sample apps
+description: Sample apps for WebView2, for various frameworks or platforms including WinUI 2 (UWP), WinUI 3 (Windows App SDK), WPF, WPF with Chrome DevTools Protocol extension, WinForms, Win32/C++, and Win32 with Visual Composition.
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 04/27/2022
+ms.date: 07/14/2022
 ---
-# Sample Code for WebView2
+# Sample apps
 
-This article is about understanding what samples, `.sln` files, and `README.md` files are in the [WebView2Samples repo](https://github.com/MicrosoftEdge/WebView2Samples).
+Sample apps that use WebView2 are available in the **WebView2Samples** repo, for various frameworks or platforms.
```

Comment:
We should add a link to the sample apps repo here: https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps